python/dlr/tf_model.py

```diff
@@ -0,0 +1,153 @@
+# coding: utf-8
+import os
+import tensorflow as tf
+
+# A prefix that will be prepended to the names in graph_def
+PREFIX = "import"
```

Comment:
would be great to handle other prefix, as well as "no prefix". We met other prefix before actually

Reply:
Non empty Prefix is needed to correctly find input and output tensors from frozen graph. I saw that in some cases frozen graphs contain lots of operators without prefix which have tensors which might be wrongly identified as global input/output tensors.